source/css/scss-common/base/_global-classes.scss

```diff
@@ -132,6 +132,14 @@ object {
     }
   }
 
+  &[x-placement='top-start'] {
+    &::after {
+      top: calc( 100% - 1px );
+      left: 40%;
+      transform: rotate(-45deg);
+    }
+  }
+
```

Comment:
have u also tested how it behaves on the large breakpoint after this fix? I tried this before and had problems in alignment for large breakpoint